### PR TITLE
Adding small scale control plane workloads for azure-aro

### PR DESCRIPTION
**Pull request by vishnuchalla**
Setting azure-aro control plane tests in PROW.
* https://docs.google.com/presentation/d/1fd23eZVqngLiYKDAhLo_odYmTpPV06vqkTUSXHhrCus/edit#slide=id.gc52980a47d_1_0
* https://docs.google.com/document/d/1_EY4FXiCht80w6ZLo6Ri56Qb2DQC5P9c3uUwEibUui0/edit

We will get infra nodes get added in a different PR!

### CODE REVIEW

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-24nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-60nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-24nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-60nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-24nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-60nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-24nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-60nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-24nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-60nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-24nodes-aro

**Comment by vishnuchalla** [comment]
az vm list-usage --location "northcentralus" --output table

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-6nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-6nodes-aro

**Comment by vishnuchalla** [comment]
@krishvoor @paigerube14 @jianlinliu PTAL 👋🏼

**Comment by jianlinliu** [comment]
For `aro-provision-cluster` step, looks good to me. cc @mgahagan73 

/lgtm

**Review by krishvoor — APPROVED** [review]
/lgtm

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-6nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-6nodes-aro

**Comment by mgahagan73** [comment]
I've reviewed the bits specific to ARO provisioning and they look fine.

/lgtm

**Comment by svetsa-rh** [comment]
@vishnuchalla For medium scale jobs, why are we going with 60 nodes? I see 120 nodes for medium scale here https://docs.google.com/document/d/1sVkbU5ttS3g_GqfEeAWDzT25asvy_jKqy0qGCJ4Lfzo/edit

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml [hunk 1]
```diff
@@ -30,15 +30,39 @@ resources:
...
   cron: 0 12 * * 3
   steps:
     cluster_profile: azure-qe
     env:
+      ARO_MASTER_VM_SIZE: Standard_D8s_v3
       ARO_WORKER_COUNT: "6"
       ARO_WORKER_VM_SIZE: Standard_D8s_v3
     test:
-    - chain: openshift-qe-cluster-density-v2
+    - chain: openshift-qe-node-density-cni
```
Comment (svetsa-rh) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:42]: @vishnuchalla Same comment. Is the intention here not to run cluster-density-v2 tests for ARO?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml [hunk 2]
```diff
@@ -30,15 +30,39 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-ci-tests-aro-cluster-density-v2
+- as: ocp-qe-perfscale-ci-tests-node-density-cni-6nodes-aro
```
Comment (svetsa-rh) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:33]: @vishnuchalla Is the intention here not to run cluster-density-v2 tests for ARO? Hence, it's deleted? 
Do we want to run only node-density tests?

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 3]
```diff
@@ -1,4 +1,79 @@
...
+    org: openshift-qe
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: azure4
+    ci-operator.openshift.io/cloud-cluster-profile: azure-qe
+    ci-operator.openshift.io/variant: aro-4.13-nightly
+    ci.openshift.io/generator: prowgen
+    job-release: "4.13"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-6nodes-aro
```
Comment (svetsa-rh) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:19]: @vishnuchalla Can we add slack notifications for all the new tests you have added? You need to update this periodics yaml file and add it below `name` and above `spec`.
Here is an example https://github.com/svetsa-rh/release/blob/master/ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml#L20

**Review by svetsa-rh — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml (hunk 2 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:33]: @vishnuchalla IMO, we should still keep cluster-density-v2 tests as they are more simplistic and reliable for catching any infra issues quicker than node-density tests.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml (hunk 1 above)
Comment (paigerube14) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:42]: the cluster-density-v2 test is in the chain in line 65

**Comment by vishnuchalla** [comment]
> @vishnuchalla For medium scale jobs, why are we going with 60 nodes? I see 120 nodes for medium scale here https://docs.google.com/document/d/1sVkbU5ttS3g_GqfEeAWDzT25asvy_jKqy0qGCJ4Lfzo/edit
> 
> For ARO, if we decided to go with 60 nodes instead, can we update the ref doc above. TIA!

@svetsa-rh As discussed over the meeting. The plan is to start with very small number i.e 6, get things working and move to a larger scale. But JFYI, current we are thinking only to do testing till 24 nodes and based on the results we can take things forward, If that sounds good.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml (hunk 2 above)
Comment (vishnuchalla) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:33]: Ack. Reg:cluster-density v2 tests, we have this [chain](https://steps.ci.openshift.org/chain/openshift-qe-ocp-qe-perfscale-ci-tests) included, that covers cluster-density-v2.

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml (hunk 3 above)
Comment (vishnuchalla) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:19]: Sure. Will get them added as suggested.

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-6nodes-aro

**Pull request merged**